File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1040/BusinessApplication/BA.Web/Models/Shared/User.shared.vb
Namespace Web

    Partial Public Class User
        ''' <summary>
        ''' Restituisce il nome utente visualizzato, che per impostazione predefinita corrisponde all'oggetto FriendlyName.
        ''' Se FriendlyName non è impostato, viene restituito l'oggetto User Name.
        ''' </summary>
        Public ReadOnly Property DisplayName() As String
            Get
                If Not String.IsNullOrEmpty(Me.FriendlyName) Then
                    Return Me.FriendlyName
                Else
                    Return Me.Name
                End If
            End Get
        End Property
    End Class
End Namespace

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1040/BusinessApplication/BA.Web.10.0/Models/Shared/User.shared.cs
namespace $safeprojectname$
{
    /// <summary>
    /// Classe parziale che estende il tipo User per aggiungere proprietà e metodi condivisi.
    /// Tali proprietà e metodi sono disponibili sia per l'applicazione server che per l'applicazione client.
    /// </summary>
    public partial class User
    {
        /// <summary>
        /// Restituisce il nome utente visualizzato, che per impostazione predefinita corrisponde all'oggetto FriendlyName.
        /// Se FriendlyName non è impostato, viene restituito l'oggetto User Name.
        /// </summary>
        public string DisplayName
        {
            get
            {
                if (!string.IsNullOrEmpty(this.FriendlyName))
                {
                    return this.FriendlyName;
                }
                else
                {
                    return this.Name;
                }
            }
        }
    }
}
